core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -191,6 +191,13 @@ class LogCleaner(initialConfig: CleanerConfig,
     */
   override def reconfigure(oldConfig: KafkaConfig, newConfig: KafkaConfig): Unit = {
```

Comment:
The javadoc should also update about updating the throttler.

Reply:
Thank you. Java doc have been updated about updating the throttler.desiredRatePerSec. https://github.com/apache/kafka/pull/12296/commits/04264960fe7300e02dd48d9f7aee66390754a417